dockerfiles/cli/build.sh

```diff
@@ -8,6 +8,8 @@
 # Contributors:
 #   Florent Benoit - Initial Implementation
 
+set -e
```

Comment:
do we need it as we include build.include ?

Reply:
not sure, but just in case I added this :)

Reply:
I think we can drop on all .sh that imports build.include

also you should add `set -u` as well in build.include

Reply:
are you sure to remove on sh? my issue was that this https://github.com/codenvy/codenvy/blob/master/dockerfiles/agents/build.sh#L16 line failed but script passed. it was before build.include was used.

you mean `. ${DIR}/../build.include`  will apply `set -e` for build.sh ?

Reply:
tested, `. ${DIR}/../build.include` will apply `set -e`

Reply:
yes

Reply:
done